r/restic/restic_ubi8.4.sh

```diff
@@ -0,0 +1,85 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : restic
+# Version       : v0.12.1
+# Source repo   : https://github.com/restic/restic/
+# Tested on     : rhel UBI 8.4
```

Comment:
If it has been tested on both RHEL and UBI, please specify both distros with versions as a comma separated list, example: "RHEL 8.4, UBI 8.4"

Reply:
Updated to UBI 8.4.